core/invoker/src/main/scala/org/apache/openwhisk/core/invoker/InvokerReactive.scala

```diff
@@ -179,6 +178,8 @@ class InvokerReactive(
     WhiskAction
       .get(entityStore, actionid.id, actionid.rev, fromCache = actionid.rev != DocRevision.empty)
       .flatMap(action => {
+        // action that exceed the system limit cannot be executed.
+        action.limits.checkSystemLimits()
```

Comment:
I think it already check the limit here? https://github.com/apache/openwhisk/pull/5229/files#diff-618ee3b5632d322d90e279346bd69b1d30f39f22b237e557f1a4f507414b61fdR216

Reply:
@jiangpengcheng The code you shared is for checking the limit when **creating an action**.
In order to prevent execution by malicious modifications such as DB alteration, the invoker side verifies once more **before invoking an action**.

Reply:
Does it apply the new scheduler? e.g. when FunctionPoolConatinerProxy fetches the activation from memoryQueue's activation

Reply:
Yes, It supports new scheduler.